### PR TITLE
Add Danbooru presence

### PR DESCRIPTION
**Pull request by kyrie25**
Few examples of how the presence works:
* Home page:
image
* Viewing artwork:
image
* Viewing user/artist:
image
image
image
* Other miscellaneous presences:
image
image

### CODE REVIEW

File: websites/D/Danbooru/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "id": "368399721494216706"
+  },
+  "service": "Danbooru",
+  "description": {
+    "en": "Danbooru is an English-based image board website specializing in erotic manga fan art."
+  },
+  "url": "danbooru.donmai.us",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/pniknY6.png",
+  "thumbnail": "https://i.imgur.com/h5Whlkk.png",
```
Comment (Slowlife01) [inline websites/D/Danbooru/dist/metadata.json:14]: Please do censor the photos

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

File: websites/D/Danbooru/dist/metadata.json (hunk 1 above)
Comment (kyrie25) [reply websites/D/Danbooru/dist/metadata.json]: Done. Hopefully I didn't miss any.

**Review by Slowlife01 — DISMISSED** [review]

**Comment by ImRodry** [comment]
Please format your file using prettier + eslint (recommended extensions)

**Comment by kyrie25** [comment]
> Please format your file using prettier + eslint (recommended extensions)

I did and ESLint showed up with no errors at the time this pull request was made. Prettier was functional also.

**Comment by ImRodry** [comment]
Nope, I am sure you did not format with prettier, check again

**Comment by kyrie25** [comment]
> Nope, I am sure you did not format with prettier, check again

@ImRodry Indeed it wasn't, just ran Prettier again, thanks for the head up!

File: websites/D/Danbooru/presence.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    clientId: "918794311557058590"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "danbooru",
+      startTimestamp: browsingTimestamp
+    },
+    shortTitle = document.title.split(/[|]/, 1)[0],
```
Comment (EncryptedDev) [inline websites/D/Danbooru/presence.ts:11]: Use array deconstructing here
```suggestion
    [shortTitle] = document.title.split(/[|]/, 1),
```

Comment (kyrie25) [reply websites/D/Danbooru/presence.ts]: Done.

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**